site/src/sphinx/server-annotated-service.rst

```diff
@@ -0,0 +1,627 @@
+======================
+
+*Armeria* provides a way to write an HTTP service using annotations. It helps a user to make his or her codes
+simple and easy to understand. A user is able to run an HTTP service by couple of lines codes using
+annotations as follows. ``getHello()`` method in the example would be mapped to the path of ``/hello``
+with an HTTP GET method.
+
+.. code-block:: java
+
+    ServerBuilder sb = new ServerBuilder();
+    sb.annotatedService(new Object() {
+        @Get("/hello/{name}")
+        public HttpResponse hello(@Param("name") String name) {
+            return HttpResponse.of(HttpStatus.OK,
+                                   MediaType.PLAIN_TEXT_UTF_8,
+                                   "Hello, %s!", name);
+        }
+    });
+
+Mapping HTTP service methods
+----------------------------
+
+To map each method in an annotated HTTP service class to HTTP path, both `@Path`_ annotation and HTTP method
+annotations are used. Here is the list of HTTP method annotations that *Armeria* supports. Each of them is
+mapped to the HTTP method which has the same name.
+
+- `@Get`_
+- `@Head`_
+- `@Post`_
+- `@Put`_
+- `@Delete`_
+- `@Options`_
+- `@Patch`_
+- `@Trace`_
+
+If you want to handle several types of HTTP methods in a single service method, you can annotate your
+service method as follows.
+
+.. code-block:: java
+
+    class MyAnnotatedServiceClass {
+        @Get
+        @Post
+        @Put
+        @Delete
+        @Path("/hello")
+        public HttpResponse hello() {
+            ...
+        }
+    }
+
+To handle a single HTTP method in a single service method, you can annotate your service method simply,
+as follows. In this case, you can omit `@Path`_ annotation and write your path with `@Get`_ annotation.
+
+.. code-block:: java
+
+    class MyAnnotatedServiceClass {
+        @Get("/hello")
+        public HttpResponse hello() {
+            ...
+        }
+    }
+
+There are 5 types of PathMapping_ s provided for describing a path with `@Path`_ annotation.
+
+- Exact mapping
+
+  - a service method will handle the path exactly matched with the specified path.
+    e.g, ``/hello`` or ``exact:/hello``
+
+- Mapping with path prefix
+
+  - a service method will handle every path which starts with the specified prefix.
+    e.g, ``prefix:/hello``
+
+- Mapping with path variables
+
+  - a service method will handle the path matched with the specified path pattern. A path variable in the
+    specified pattern may be mapped to a parameter of the service method.
+    e.g, ``/hello/{name}`` or ``/hello/:name``
+
+- Mapping with regular expression
+
+  - a service method will handle the path matched with the specified regular expression. If a named capturing
+    group exists in the regular expression, it may be mapped to a parameter of the service method.
+    e.g, ``regex:^/hello/(?<name>.*)$``
+
+- Mapping with glob pattern
+
+  - a service method will handle the path matched with the specified glob pattern.
```

Comment:
Could you mention that each component will be mapped to the parameter whose name is an integer? (with example, please)